tests/unit/test_pskc.cpp

```diff
@@ -89,12 +90,12 @@ int main(void)
     return 0;
```

Comment:
What about adding here a test for the example test vector provided in the Thread Spec section 8.4.1.2.2?